### ISSUE TITLE
OAS 3.0: Parameter types are not displayed

### ISSUE THREAD

**Issue by hkosova** [swagger-api/swagger-ui#3428]
Version: `ft/oas3` branch, as of commit a1ce0e7b3c54d66534faf59599c6bce5f23656f2

Parameter types (integer, string, etc.) are not displayed for OAS 3.0 specs:

![parameter-types](https://user-images.githubusercontent.com/8576823/28457307-420aca88-6e0e-11e7-9017-d4a867144629.png)

They are displayed for 2.0 specs:
![parameter-types 2.0](https://user-images.githubusercontent.com/8576823/28457489-f6627ad0-6e0e-11e7-9895-5a748c3abb6a.png)

Sample spec:
```
openapi: 3.0.0
info:
  version: 0.0.0
  title: test
paths:
  /{id}:
    get:
      parameters:
        - in: path
          name: id
          required: true
          schema:
            type: integer
        - in: query
          name: foo
          schema:
            type: boolean
        - in: header
          name: X-Something
          schema:
            type: string
      responses:
        '200':
          description: OK
```

@shockey

**Comment by hkosova** [comment]
Here's another example with an `object`-type parameter. How are object schemas supposed to be displayed?

```
openapi: 3.0.0
info:
  version: 0.0.0
  title: test

servers:
  - url: http://httpbin.org

paths:
  /anything/foo{params}:
    get:
      parameters:
        - in: path
          name: params
          required: true
          schema:
            type: object
            properties:
              lat:
                type: integer
                example: 50
              long:
                type: integer
                example: 20
              scale:
                type: integer
                example: 32000
          style: matrix
          explode: true
      responses:
        '200':
          description: OK
```

**Comment by webron** [comment]
@hkosova they're supposed to be displayed the same as request bodies.

**Comment by shockey** [comment]
Fixed first part of this in #3557 - moving the object parameter schema display to a new ticket.
